### DISCUSSION TITLE
LSP installation always gets stuck.

### DISCUSSION THREAD

**Discussion by txtyash** [mason-org/mason.nvim#1371] [General]
I installed lazyvim and the language servers don't get installed they just get stuck during install
![image](https://github.com/williamboman/mason.nvim/assets/85027668/ce165e6a-fafb-4e7d-a104-06adeb7a0205)

Reference:
https://github.com/LazyVim/LazyVim/discussions/988
& also this didn't work for me: https://www.reddit.com/r/neovim/comments/1405d71/mason_stuck_while_installing_lsp/

**Comment by williamboman** [comment]
Hello! This seems similar to https://github.com/williamboman/mason.nvim/issues/1361. It appears related to the download of the LSP configuration schema which often downloads from `raw.githubusercontent.com`. This is not necessary for the package to function, so I've added a 5s timeout [here](https://github.com/williamboman/mason.nvim/pull/1374) which hopefully allows you to proceed with the installation.

**Reply by txtyash** [reply]
Yup, svelte server was successfully installed. Thanks!
